### DISCUSSION TITLE
Scraping le boncoin et se loger

### DISCUSSION THREAD

**Discussion by MAHRAZ-Oussama** [community/community#144917] [Programming Help]
Bonjour à tous,

Je travaille sur un projet visant à centraliser les annonces immobilières provenant de plateformes comme LeBonCoin et SeLoger. L'idée est de créer un moteur de recherche permettant de naviguer facilement entre les annonces, avec des critères avancés comme le prix, la localisation, la surface, etc.

Objectifs principaux :
Extraire les données : via API (quand disponible) ou scraping HTML.
Agréger les annonces : regrouper les résultats des deux plateformes tout en gérant les doublons.
Offrir une navigation fluide : avec des recherches multi-critères et un système de tri intelligent.

**Comment by mecodeatlas** [comment]
Hi @MAHRAZ-Oussama, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
